sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveShowCreateTableSuite.scala

```diff
@@ -19,36 +19,19 @@ package org.apache.spark.sql.hive
 
 import org.apache.spark.sql.{AnalysisException, ShowCreateTableSuite}
 import org.apache.spark.sql.hive.test.TestHiveSingleton
+import org.apache.spark.sql.internal.SQLConf
 
 class HiveShowCreateTableSuite extends ShowCreateTableSuite with TestHiveSingleton {
 
   test("simple hive table") {
-    withTable("t1") {
-      sql(
-        s"""CREATE TABLE t1 (
-           |  c1 INT COMMENT 'bla',
-           |  c2 STRING
-           |)
-           |TBLPROPERTIES (
-           |  'prop1' = 'value1',
-           |  'prop2' = 'value2'
-           |)
-         """.stripMargin
-      )
-
-      checkCreateTable("t1")
-    }
-  }
-
-  test("simple external hive table") {
-    withTempDir { dir =>
+    withSQLConf(
+        SQLConf.LEGACY_CREATE_HIVE_TABLE_BY_DEFAULT_ENABLED.key -> "true") {
```

Comment:
legacy configs may get removed in future versions. Can we update the testing query to add `USING hive` or `STORE AS TEXT`?

Reply:
There is a test `simple hive table`. The purpose of the test should be a very simple `CREATE TABLE` query.  My original proposal is just to add legacy config enabled for few tests like this one.

Adding `USING hive` is ok, I'm just concerned that it will reduce test coverage.

Reply:
I think as @HyukjinKwon said we can use beforeAll. WDYT?

Reply:
+1 for `beforeAll`.